General/general_shacl.ttl

```diff
@@ -0,0 +1,131 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:GeneralShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node gx:DataResource ;
+            sh:order 1 ;
+            sh:path general:DataResource ],
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:LinksShape ;
+            sh:order 4 ;
+            sh:path general:Links ] ;
+    sh:targetClass general:General .
+
+general:LinksShape a sh:NodeShape ;
+    sh:property [ skos:example "link to screenshot, video, routing" ;
+            sh:description "Reference to screenshot data"@en ;
+            sh:message "Validation of media failed!"@en ;
+            sh:name "media"@en ;
+            sh:node general:LinkShape ;
+            sh:order 2 ;
+            sh:path general:media ],
+        [ skos:example "???" ;
+            sh:datatype xsd:string ;
+            sh:description "contract id -> is generated/assigned later via the EDC"@en ;
+            sh:maxCount 1 ;
+            sh:message "Validation of contractID failed!"@en ;
+            sh:name "contractID"@en ;
+            sh:order 3 ;
+            sh:path general:contractID ],
+        [ skos:example "link to CRG" ;
+            sh:description "Reference to dependent data"@en ;
+            sh:message "Validation of data failed!"@en ;
+            sh:minCount 0 ;
+            sh:name "data"@en ;
+            sh:node general:LinkShape ;
+            sh:order 1 ;
+            sh:path general:data ];
+    sh:targetClass general:Links .
+
+general:LinkShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:string ;
+            sh:in ("Document" "Image" "Model" "Routing" "Video") ;
+            sh:message "Validation of type failed!"@en ;
+            sh:description "Choose type of link." ;
+            sh:name "type"@en ;
+            sh:order 1 ;
+            sh:path general:type ],
+        [ sh:datatype xsd:anyURI ;
+            sh:message "Validation of url failed!"@en ;
+            sh:description "Enter link as URL." ;
+            sh:name "url"@en ;
+            sh:order 0 ;
+            sh:path general:url ] ;
+    sh:targetClass general:Link .
+
+gx:DataResource a sh:NodeShape ;
```

Comment:
DataResource not necessary here.
DataResource will be the superclass of HD-Map (or HDMap subclass of DataResource), Scenario etc. So no link in the general section necessary.

Reply:
then please copy gx:DataResource into the data Shacl's. But then there is almost nothing left in General, except DataRef and Media links. Please discuss together whether Genral is still worthwhile - also with regard to several Shacl - several claims

Reply:
gx:DataResource should also not be in the DataShacls.

It would be fine for me if General has only a few attributes. The less the better.
@jtdemer We need a decission on how to proceed here. Any opinions in the DataProvider's group if they want to keep General or not?

Reply:
remove gx:DataResource